# -*- coding: utf-8 -*-
# Generated by Django 1.10.7 on 2017-07-25 20:05


import django.contrib.auth.validators
from django.db import migrations, models


class Migration(migrations.Migration):

    dependencies = [
        ('badgeuser', '0012_auto_20170711_1326'),
    ]

    operations = [
        migrations.AlterField(
            model_name='badgeuser',
            name='username',
            field=models.CharField(error_messages={'unique': 'A user with that username already exists.'}, help_text='Required. 150 characters or fewer. Letters, digits and @/./+/-/_ only.', max_length=150, unique=True, validators=[django.contrib.auth.validators.ASCIIUsernameValidator()], verbose_name='username'),
        ),
    ]
